.rubocop.yml

```diff
@@ -121,22 +292,22 @@ Layout/SpaceInsideBlockBraces:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
 
+# enforces that parentheses do not have spaces
 Layout/SpaceInsideParens:
   Enabled: true
 
-# Check quotes usage according to lint rule below.
+# Enforce string literals to use double quotes everywhere
 Style/StringLiterals:
-  Enabled: false
+  Enabled: true
   EnforcedStyle: double_quotes
 
-# Detect hard tabs, no hard tabs.
-Layout/IndentationStyle:
+# Requires a single final new line to the file
+Layout/TrailingEmptyLines:
   Enabled: true
+  EnforcedStyle: final_newline
```

Comment:
@yedhink I've seen rubocop complain that final new line is not there. If rubocop can automatically fix it then this is fine. If rubocop can't fix it then I find it unnecessary to manually add a blank line and in that case I would insist on disabling this cop.